examples/query/react/optimistic-update/public/manifest.json

```
@@ -0,0 +1,8 @@
+{
+  "short_name": "Simple Query Demo",
+  "name": "RTK Simple Query Demo",
```

Uhh we should change the internal CodeName here, even though it's kinda funny at this point :sweat_smile: 
```suggestion
  "short_name": "RTK Query Demo",
  "name": "RTK Query Demo",
```